datalad/config.py

```diff
@@ -129,7 +129,7 @@ def _get_default_file_candidates(self):
             cfg_file_candidates.append(opj(home_cfg_base_path, 'datalad.cfg'))
 
         # current dir config
-        cfg_file_candidates.append(opj('.datalad', 'config'))
+        cfg_file_candidates.append('datalad.cfg')
```

Comment:
Yes.

On Tue, Aug 30, 2016 at 3:27 PM, Yaroslav Halchenko <
notifications@github.com> wrote:

> In datalad/config.py
> <https://github.com/datalad/datalad/pull/744#discussion_r76793811>:
>
> > @@ -129,7 +129,7 @@ def _get_default_file_candidates(self):
> >              cfg_file_candidates.append(opj(home_cfg_base_path, 'datalad.cfg'))
> >
> >          # current dir config
> > -        cfg_file_candidates.append(opj('.datalad', 'config'))
> > +        cfg_file_candidates.append('datalad.cfg')
>
> ah right... since we are aiming for unification and don't think anyone
> would have those datalad.cfg laying around -- just comment it out entirely
> I guess. later it should read .datalad/config right?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/datalad/datalad/pull/744/files/65570c6d158c41afcab243aca653d2b3b08e833e#r76793811>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAIVH8Vn2ODoeYD5mvcJOnYKgynbPCKyks5qlC_DgaJpZM4JwXjp>
> .
>



-- 
Michael Hanke
http://mih.voxindeserto.de